src/components/c-google-maps.vue

```diff
@@ -162,20 +162,19 @@
         throw new Error('No Google Maps API key provided.');
       }
 
-      const url = `https://maps.googleapis.com/maps/api/js?key=${apiKey}&libraries=places`;
-
       /**
        * Callback for the load event of the map script.
        */
-      const callback = () => {
+      window.initMap = () => {
```

Comment:
@patric-eberle  I understand, but there is an error caused by the use of const in the declaration of window.cGoogleMapsInitMap.

`const' variable without initializer is not allowed`

Reply:
@patric-eberle 
> @BeneRichi And we need to test if still multiple maps can be initialized on the same page with the new callback method (in the URL).

-> Unfortunately No, it is not working anymore with the new callback. 

This could be a workaround:

`window.dummy = () => {};

const url = `https://maps.googleapis.com/maps/api/js?key=${this.getGoogleMapsApiKey}&libraries=places&callback=dummy`;`